_posts/2016-07-04-Test-Your-Swagger-API.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Test Your Swagger API
+excerpt: Introducing support for Swagger in Dredd and more.
+layout: post
+date: 2016-07-04 00:00:00 +0100
+author: honza
+published: false
+comments: true
+---
+
+It's quite common knowledge that in Apiary we're proponents of design-first approach when it comes to building APIs. However, it's hard to do design-first and have your "single source of truth" in the API description document, if you can't easily test your API implementation against it.
+
+## Support for Swagger in Dredd
+
+It's a [long time][original-dredd-article] since we announced [Dredd][dredd], an Open Source framework for testing HTTP-based APIs. Over the years it grown into a fully-featured, mature, and very popular tool. The main flaw remaining was the fact that Dredd was able to read only [API Blueprint][api-blueprint] format.
+
+Now that's not true anymore! With 1.1.0 release, all Dredd's features are now available also for users of the [Swagger][swagger] API description format. Since version 1.5.0 the Swagger support is considered production-ready. Here we would like to thank to all the adventurous minds who helped us to improve the initial implementation. Only thanks to you we could remove the "experimental" label so soon!
```

Comment:
Change the tone 

> The main flaw remaining

it wasn't a flaw. that was the design of the tool. Turn it around. E.g until now it supported only API Blueprint, but we want empower everybody to test their API regardless of the chosen description format..